composer.json

```diff
@@ -3,10 +3,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.0",
-        "phpunit/phpunit-mock-objects": "^5.0.4|^6.0.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^6.5|^7.0"
+        "phpunit/phpunit": "^6.0|^7.0|^8.0"
```

Comment:
are you sure it still works with 6.0 like that?

Reply:
i can add travis settings, but you would need to register it

Reply:
I gladly will do that. 😃

Reply:
done

Reply:
i think 6.5 is minimal (used 6.5 in my lib for that reason)